src/mysql-util/src/bin/mysql-gen.rs

```diff
@@ -0,0 +1,200 @@
+// Copyright Materialize, Inc. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+//! Insert random data into MySQL, assuming the Debezium demo database
+
+use chrono::Utc;
+use env_logger::{Builder as LogBuilder, Env, Target};
+use log::{error, info};
+use mysql::prelude::Queryable;
+use mysql::{Pool, TxOpts};
+use rand::seq::SliceRandom;
+use rand::{thread_rng, Rng};
+use std::thread::{sleep, spawn, JoinHandle};
+use std::time::Duration;
+
+type Error = Box<dyn std::error::Error + Send + Sync>;
+type Result<T> = std::result::Result<T, Error>;
```

Comment:
I'm not sure this is written down anywhere, but general consensus is that these type aliases make things more confusing, and we prefer to spell out `Result<T, Box<dyn Error>>` where necessary. Also, `Box<dyn Error>` is generally more conveniently replaced with `anyhow::Error`.

Reply:
Sure -- I can do that! I had copied this style from `src/peeker/src/main.rs` and can avoid doing that if the style has changed.

Reply:
Ah yeah, Peeker is out of date it seems.